### PR TITLE
Increment version

### PR DESCRIPTION
**Pull request by alexathomases**
## Description

This PR updates the HealthDataAIServices module to use our stable API version (2024-09-20).

## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/HealthDataAIServices/HealthDataAIServices.Autorest/README.md [hunk 1]
```diff
@@ -41,7 +40,7 @@ try-require:
   - $(repo)/specification/healthdataaiservices/resource-manager/readme.powershell.md
  
 # For new RP, the version is 0.1.0
-module-version: 0.1.0
+module-version: 1.0.0
```
Comment (VeryEarly) [inline src/HealthDataAIServices/HealthDataAIServices.Autorest/README.md:43]: this line of change is unnecessary
```suggestion
module-version: 0.1.0
```

**Review by VeryEarly — DISMISSED** [review]

**Comment by alexathomases** [comment]
@VeryEarly I followed the guidance not to manually change the version in the README.md. How can I ensure that the version gets updated to 1.0.0 to indicate that this is a stable release, not preview?

**Comment by VeryEarly** [comment]
to become a stable version, we required preview version of module to be released for at least 2 months. This change will be published as 0.2.0 this time. Please ask for GA early next year.

**Comment by alexathomases** [comment]
> to become a stable version, we required preview version of module to be released for at least 2 months. This change will be published as 0.2.0 this time. Please ask for GA early next year.
> 
> Please also provide release notes in comment of this PR

That makes sense - thank you. What is the process to ask for GA?
Added description in PR comment.

**Comment by VeryEarly** [comment]
Hi @alexathomases , please refer to https://eng.ms/docs/cloud-ai-platform/azure-core/azure-experiences-and-ecosystems/azure-portal-and-client-tools-ruhim/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/azure_client_tools/onboarding/azurepowershell/release

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**